### DISCUSSION TITLE
Pulling SpoolmanDB into Spoolman..

### DISCUSSION THREAD

**Discussion by sntlewis** [Donkie/Spoolman#530] [Q&A] [answered]
OK, I'm dense on this matter.  How do I pull SpoolManDB data into my Spoolman database?  I'm just not getting it.

**Comment by Donkie** [comment, marked as answer]
SpoolmanDB is loaded automatically on startup. You will see the data from this database when you go to create a new spool and select filament from the list. In the list, you will be able to select both from internal and external filaments. External filaments are taken from SpoolmanDB.

**Comment by sntlewis** [comment]
Well I'll be.  Very grateful for the graceful answer.
